### PR TITLE
Java 8 updates

### PR DESCRIPTION
**Pull request by xandris**
Minimal testing on my part to see if the UI works. The new UI looks nice!

### CODE REVIEW

File: plugins/database-migration-1.4.0/LICENSE.txt~d02bb5bc58476325dd3fe88139bb5bd8ca2d3bec [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (rpokorny) [inline plugins/database-migration-1.4.0/LICENSE.txt~d02bb5bc58476325dd3fe88139bb5bd8ca2d3bec]: Should this file have been committed?  It looks like a backup or merge artifact.

File: plugins/database-migration-1.4.0/LICENSE.txt~Update plugins [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (rpokorny) [inline plugins/database-migration-1.4.0/LICENSE.txt~Update plugins]: Should this file have been committed? It looks like a backup or merge artifact.

**Comment by stephanieerin** [comment]
Couple of things: I managed to get a bundle but I had to disable the automated tests to get around 5 of them failing. I also was unable to get this running in developer mode (`grails run-app -https -Duser=testAdmin1`) because of a dependency's incompatibility with Java 8 (looks like KeyTool).

Let me know if you want to divvy up any of this, I'm happy to take a look at some of it.

File: plugins/database-migration-1.4.0/LICENSE.txt~Update plugins (hunk 2 above)
Comment (xandris) [inline plugins/database-migration-1.4.0/LICENSE.txt~Update plugins]: Those should probably be removed

**Comment by xandris** [comment]
I pushed a fix for that KeyTool error.

**Comment by stephanieerin** [comment]
I don't know how easy this is to do but could you submit this PR to the new branch I created called `java-8`? It might involve just closing this one and opening a new one if you can't edit an existing one.

**Pull request closed without merging**